tests/test_element.py

```diff
@@ -42,3 +42,7 @@ def test_element_type() -> None:
 
     assert_type(div()["a"], Element)
     assert isinstance(div()["a"], Element)
+
+
+def test_del_keyword_element() -> None:
+    assert str(del_) == "<del></del>"
```

Comment:
could you also add a test for an element that is not part of the predefined list? continue? class or similar?

Reply:
Done!